inventory/sample/group_vars/all.yml

```diff
@@ -40,6 +40,10 @@ extra_server_args: >-
 extra_agent_args: >-
   {{ extra_args }}
 
+extra_worker_packages: []
+# - open-iscsi  # for longhorn
+# - nfs-common  # for longhorn backups
+
```

Comment:
This variable are instantiated one time for every Ansible host, so I think a better approach to this feature would be to have an `extra_packages` variable that is set for specific hosts or groups (in this case, the equivalent would be the `node` group). That would give you a lot more flexibility (e.g. when wanting to install packages on control nodes, dealing with different distros, etc.)

Reply:
Is this as easy as defining another var file with the right name? Think this is an interesting approach, just not totally sure what it would look like